week4_2/src/pages/login.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { Link } from "react-router-dom";
+import Contentbox from "../components/contentBox";
+import Title from "../components/Title";
+import InputForm from "../components/inputForm";
+import Cta from "../components/cta";
+import ToastMessage from "../components/toastMessage";
+import apiClient from "../api/axios";
+
+const Login = () => {
+  const [idValue, setIdValue] = useState("");
+  const [pwValue, setPwValue] = useState("");
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState("");
+
+  const handleToastClose = () => {
+    setShowToast(false);
+  };
+  const navigate = useNavigate();
+
+  //id 비밀번호
+  const handleLogin = async () => {
+    try {
+      const response = await apiClient.post(`/api/v1/members/sign-in`, {
+        username: idValue,
+        password: pwValue,
```

Comment:
애초에 state를 각각
const [userName, setUserName] = useState("");
  const [password, setPassWord] = useState("");
이렇게 하면 
```suggestion
        username,
        password,
```

단축 프로퍼티라는 것에 의해 이렇게도 가능하답니당? 그냥 참고!!